.pre-commit-config.yaml

```diff
@@ -60,3 +60,10 @@ repos:
     rev: v2.10
     hooks:
       - id: vulture
+  - repo: local
+    hooks:
+      - id: generate-manpages
+        name: Generate manpages
+        entry: scripts/manpage_generation.sh
+        language: script
+        files: 'convert2rhel/__init__.py|specfile'
```

Comment:
We might just wanna wrap the manpage generation in a container if possible so it works on all machines

Reply:
Ok, I can do that! I'll work on that and get it pushed back up to this PR!